lib/cm_admin/view_helpers/field_display_helper.rb

```diff
@@ -26,6 +28,21 @@ def show_rows(ar_object, rows, col_size: 3)
         end.join.html_safe
       end
 
+      def create_alerts(alerts, ar_object)
+        alerts.map do |alert|
+          create_alert(alert, ar_object)
+        end.join.html_safe
+      end
+
+      def create_alert(alert, ar_object)
+        return unless alert.display_if.call(ar_object)
+
+        alert_icon_info = { warning: 'triangle-exclamation', danger: 'triangle-exclamation', success: 'circle-check' }
```

Comment:
```suggestion
        alert_icon_info = { warning: 'triangle-exclamation', danger: 'triangle-exclamation', success: 'circle-check' }
```

Is there a reason why this is called `alert_icon_info`? From what I see, it is the defaults based on the type of alert set

Reply:
@mikevic I believe the keys are based on the alert type and used.